### PR TITLE
contrib, cgroup v2: use kubernetes master

### PR DESCRIPTION
**Pull request by giuseppe**
Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind dependency-change
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by giuseppe** [comment]
updated to use kubernetes master

/hold cancel

File: contrib/test/integration/main.retry [hunk 1]
```diff
@@ -0,0 +1 @@
+root@192.168.125.111
```
Comment (haircommander) [inline contrib/test/integration/main.retry:1]: what is this?

File: test/pod.bats [hunk 2]
```diff
@@ -336,7 +336,7 @@ function teardown() {
 
 @test "kubernetes pod terminationGracePeriod passthru" {
 	[ -v CIRCLECI ] && skip "runc v1.0.0-rc11 required" # TODO remove this
-	CONTAINER_CGROUP_MANAGER="systemd" start_crio
+	CONTAINER_CGROUP_MANAGER="systemd" XDG_RUNTIME_DIR="" start_crio
```
Comment (haircommander) [inline test/pod.bats:339]: I think we should have an explaination on why we need this change in a comment

**Comment by haircommander** [comment]
:tada: so exciting :smile:

File: contrib/test/integration/main.retry (hunk 1 above)
Comment (giuseppe) [reply contrib/test/integration/main.retry:1]: garbage :) I'll drop it

File: test/pod.bats (hunk 2 above)
Comment (giuseppe) [reply test/pod.bats:341]: sure, added

**Comment by haircommander** [comment]
/hold

what's this file
` contrib/test/integration/callback_plugins/__pycache__/default.cpython-37.pyc `

(it's binary, so i can't comment on it)

**Comment by giuseppe** [comment]
> what's this file
> `contrib/test/integration/callback_plugins/__pycache__/default.cpython-37.pyc`
> 
> (it's binary, so i can't comment on it)

sorry I thought I cleaned up the PR but there was still an extra file.  Thanks for spotting it.

**Comment by haircommander** [comment]
holy green CI
:tada: thanks @giuseppe 

/hold

**Comment by giuseppe** [comment]
> my goodness, green CI
> tada thanks @giuseppe
> 
> /hold

I need to drop the last patch before we can merge

**Comment by giuseppe** [comment]
> I need to drop the last patch before we can merge

I dropped it.  We can restart the CI once the Kubernetes PR is merged

**Comment by giuseppe** [comment]
/hold cancel

@haircommander finally it is ready

**Comment by haircommander** [comment]
/lgtm

:tada:

**Comment by TomSweeneyRedHat** [comment]
LGTM
once the tests are hip

**Pull request merged**